src/Database/Type.php

```diff
@@ -145,8 +145,12 @@ public static function set($name, Type $instance)
      * If called with no arguments it will return current types map array
      * If $className is omitted it will return mapped class for $type
      *
-     * Deprecated: The usage of $type as \Cake\Database\Type[] is deprecated. Please always use string[] if you pass an array
-     * as first argument.
+     * Deprecated:
```

Comment:
Do we know the version when it was deprecated and the new methods added? This will help when having to refactor plugins and to know what min requirement to specify. We should add it here then.